src/ray/gcs/gcs_server/gcs_task_manager.h

```diff
@@ -255,6 +262,9 @@ class GcsTaskManager : public rpc::TaskInfoHandler {
     /// A iterator into task_events_ that determines which element to be overwritten.
     size_t next_idx_to_overwrite_ = 0;
 
+    /// Total number of task_events, including the ones have been overwritten.
+    size_t total_num_events_ = 0;
```

Comment:
```suggestion
    size_t total_num_tasks_ = 0;
```

Since this is tracking on per task level. A task could have multiple events reported.